docs/enterprise-guide/sso-keycloak.md

```diff
@@ -0,0 +1,59 @@
+# Enabling SSO with Keycloak
```

Comment:
@paoliniluis I took this from the instructions you wrote up. Let me know if it's correct, and whether it's still clear without the screenshots!

Reply:
thanks!